app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,5 @@
     <string name="app_name">SampleApp</string>
     <string name="title_activity_main">MainActivity</string>
     <string name="title_activity_second">SecondActivity</string>
+    <string name="web_client_id">475882605501-gv47hlfhkd5j459vjv0kc1cj97ensm15.apps.googleusercontent.com</string>
```

Comment:
I am not sure it is ideal to place web_client_id directly in the strings.xml file means it will be part of our app's compiled resources, which could be decompiled and accessed by anyone I guess